euphonic/qpoint_phonon_modes.py

```diff
@@ -53,8 +53,8 @@ def __init__(self, crystal, qpts, frequencies, eigenvectors, weights=None):
         eigenvectors: (n_qpts, 3*crystal.n_atoms, crystal.n_atoms, 3) complex ndarray
             Dynamical matrix eigenvectors
         weights : (n_qpts,) float ndarray, optional
-            The weight for each q-point. If None, equal weights are
-            assumed
+            The weight for each q-point, should sum to 1. If None, equal
+            weights are assumed
```

Comment:
This requirement could be a pain-point when integrating with other codes: a lot of codes use integer weights and then normalise (or don't...). Is there a technical reason we'd prefer not relax this and do our own normalisation of weights?

Reply:
That's true, I guess provided we ensure weights are normalised it doesn't really matter which convention is used, I just chose summing to 1 because that's what CASTEP uses (and actually when reading from Phonopy to a Euphonic object it converts from the integer convention to summing-to-1 convention for consistency). I'll check that everything that uses weights is normalised and remove the comment about summing to 1.